eggbox_potential_sampler/sampling_data_view/tests/test_sampling_data_view.py

```diff
@@ -48,14 +48,13 @@ def setUp(self):
         self.analysis_model = AnalysisModel()
         self.data_view = SamplingDataView(analysis_model=self.analysis_model)
         self.plot = self.data_view.sampling_plot
-        self.analysis_model.value_names = ("x", "y", "E")
+        self.analysis_model.header = ("x", "y", "E")
```

Comment:
Since this plugin is the "guideline" for the FORCE plugin developments, please let me be boring here, and suggest a change to the way we test the analysis model here.

If I remember correctly, the `AnalysisModel` implements the following API to add data to the model:
```
header = ("a", "b", "c")
data = ((1, 2, 3), (4, 5, 6))
self.model.notify(header)
for entry in data:
    self.model.notify(entry)
```
The `notify` method is preferred over setting the attributes (such as `header`) directly.

Reply:
Awesome, thank you for pointing this out - I had forgotten about it from the WfManager PR